docs/conops.md

```diff
@@ -0,0 +1,549 @@
+
+**Rideshare and Cargo Services** elements are client-facing and not safety-critical.
+
+Description of the individual services is covered in section 3.2.
+
+## 2.0 Documents
+### 2.1 Applicable Resources
+
+| Resource | Type |
+| --- | --- |
+| [UAM Airspace Research Roadmap](https://ntrs.nasa.gov/citations/20210019876) | Website
+| [NASA Air Traffic Management (ATM-X)](https://www.nasa.gov/aeroresearch/programs/aosp/atm-x/atm-x-project-description) | Website
+| [NASA Extensible Traffic Management (xTM) Research ](https://ntrs.nasa.gov/citations/20210025112) | Website
+
+## 3.0 Description of Envisioned System
+
+### 3.1 Needs, Goals and Objectives of Envisioned System
+Each service in this architecture is associated with a set of requirements.
+
+Each service has its own set of documents which should be referred to for more detailed information. These documents include:
+- Concept of Operations (CONOPS)
+- Software Design Document (SDD)
+- Interface Control Document (ICD)
+- Verification and Validation (V&V) Document
+
+A set of user stories for rideshare and cargo operations is currently tracked on the [Arrow Google Drive](https://docs.google.com/spreadsheets/d/1Ad238NAEj6QUzgsjPTRRFJy6NiQVQt2e7affwVVDAFo/edit?usp=sharing).
+
+### 3.2 Overview of System and Key Elements
+
+#### Provider of Services (PSU) Elements
+
+These services enable UAM operators to access airspace legally, safely, and securely.
+
+<center> 
+
+Service | Responsibilities
+--- | ---
+`svc-scheduler` | Dynamic airspace management<br>Vertiport scheduling<br>Flight plan bookkeeping<br>Fleet routing optimization
+`svc-discovery` | Interaction with other PSUs<br>Adheres to standardized PSU interfaces
+`svc-compliance` | Notifications, flight restrictions, authorization approvals from local authority<br>Automated dispatch of flight plans and release requests to local authority<br>Complies with regulatory file formats such as FIDXP<sup>*</sup>
+`svc-guidance` | Noncompulsory flight path guidance<br>Importance increases with number of aerial actors
+
+<sup>*</sup> FAA-Industry Data Exchange Protocol
+
+</center> 
+
+#### Supplemental Data and Service Provider (SDSP) Elements
+
+These elements provide *supplemental* (not safety-critical) services for UAM operations.
+
+<center> 
+
+Service | Responsibilities
+--- | ---
+`svc-storage` | Abstracted storage interface for the other services<br>Logs, maintenance records, certification records, and so on
+`svc-telemetry` | Receives and stores live vehicle telemetry<sup>*</sup><br>Rebroadcasts to ground control/air traffic control tools and other listeners
+`svc-upkeep` | Monitor expiration of vehicle and component certifications<br>Preemptively schedule flights for maintenance
+
+</center>
+
+<sup>*</sup> May qualify as a PSU element.
+#### Rideshare and Cargo Services
+
+These client-facing services expose an API. These are not safety-critical.
+
```

Comment:
ibid re the safety language.

Reply:
https://github.com/Arrow-air/se-services/pull/2/commits/fb227c5a17a327041869503558fcb729308c9955